core/platform/users/gae_current_user_services.py

```diff
@@ -37,33 +37,3 @@ def create_login_url(slug):
     return users.create_login_url(
```

Comment:
Should we be trying to get rid of this function, too?

Reply:
Yes, but I don't think it belongs in auth services. This will have to be deleted on its own when we switch over to firebase.

Reply:
OK, maybe add a TODO and issue?

Reply:
@brianrodri I think you missed this.